modules/@apostrophecms/schema/ui/apos/components/AposInputString.vue

```diff
@@ -121,17 +121,30 @@ export default {
           return 'required';
         }
       }
-      if (this.field.min) {
-        if (this.type === 'number' || this.type === 'date') {
+
+      const minMaxNumFields = [
+        'date',
+        'integer',
+        'float',
+        'range'
+      ];
+
+      const minMaxFields = [
+        ...minMaxNumFields,
+        'string'
```

Comment:
We should limit `min/max` use more. Emails shouldn't have min/max limitations. Nor URLs. Maybe passwords and slugs? I'd rather start more limited.

Reply:
I see no reason to forbid min on passwords or slugs. Clearly max on passwords is kind of crappy, but the field is legitimate for uses other than our own passwords and they may have constraints and it doesn't serve the user to let them enter something that won't work with system X (that the field is being passed to) if we can help them by rejecting it. So I think passwords should support both min and max, and slugs too.

Reply:
Actually, slugs are less obvious. In the field's `convert` function we do a certain amount of mutation to the slug. We run `slugify` as well as adding and removing slashes if it's a page. Do we match the min/max lengths before or after all of that?

It's not likely that min/max would be used with the page option, but even with the basic version you could be converting multiple spaces and punctuation to a single dash.

Reply:
Good points. I would apply min/max to slugs at the end.